core/contracts/oracle/implementation/Voting.sol

```diff
@@ -386,7 +386,7 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface {
         require(voteSubmission.commit != bytes32(0), "Invalid hash reveal");
         require(
             keccak256(abi.encodePacked(price, salt, msg.sender, time, roundId, identifier)) == voteSubmission.commit,
-            "Revealed data != commit hash"
+            "Revealed data does not match commit hash contents"
```

Comment:
Would recommend against making this change because the on-chain version's error message will not match. IMO, that could be more confusing than the `!=`.

Reply:
Bummer, but agreed

Reply:
done

Reply:
also we need to keep the require messages less than 32 bytes to achieve constant gas usage in require messages.